morpcc/navigator.py

```diff
@@ -65,14 +65,14 @@ def __init__(self, schema, entity, request):
 
     def add_attribute(
         self,
-        name,
-        type_,
-        title,
-        description=None,
-        required=False,
-        primary_key=False,
+        name: str,
+        type_: typing.Union[int, str, float],
+        title: str,
+        description: str = None,
```

Comment:
whenever a string variable defaults to None, the type should be typing.Optional[str]. Similarly on other fields.
```suggestion
        description: typing.Optional[str] = None,
```